plugins/Installation/templates/_systemCheckSection.twig

```diff
@@ -1,4 +1,8 @@
 {% import _self as local %}
+<script type="text/javascript" src="plugins/Morpheus/javascripts/piwikHelper.js"></script>
+<script>
+    piwik.helper = piwikHelper;
```

Comment:
Instead of defining here `piwik.helper` it would be better to change the `piwik.helper.sendContentAsDownload` to `piwikHelper.sendContentAsDownload`. That's because outside angular we always refer to it as `piwikHelper` vs in angular we usually load a dependency `piwik` which has `piwik.helper` defined. bit hard to explain but it should be basically `piwikHelper` in the download button.

Reply:
OK, sorry, had a feeling about piwikHelper and then checked the issues and you did say.

Fixed now.